Assets/Scripts/Customization/HairCustomization.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[ExecuteInEditMode]
+public class HairCustomization : MonoBehaviour 
+{
+    public static HairCustomization instance;

```

Comment:
Why is this singleton? I don't see it used anywhere

Reply:
was used in a previous version forgot to remove (got rid of it now)